hles/transformation/src/main/scala/org/broadinstitute/monster/dap/dog/ResidentialEnvironmentTransformations.scala

```diff
@@ -65,9 +65,9 @@ object ResidentialEnvironmentTransformations {
           // there are two separate fields to check for "First country"
           dePastResidenceCountry1 =
             if (pastCountryCount > 1)
```

Comment:
it won't let me make a suggestion that spans deleted lines, so don't click auto-accept, but this is my suggestion for simplifying this field's logic

```suggestion
          dePastResidenceCountry1 =
            if (pastCountryCount == 1) rawRecord.getOptional("de_country_01_only_dd") else getPastResidenceCountry(rawRecord, pastCountryCount, 1L, "de_country_01_dd")
```